subprojects/docs/src/docs/release/notes.md

```diff
@@ -26,9 +39,9 @@ See the User guide section on the “[Feature Lifecycle](userguide/feature_lifec
 
 The following are the features that have been promoted in this Gradle release.
 
-<!--
-### Example promoted
--->
+### Dependency insight report
+
+The dependency insight report is now considered stable.
```

Comment:
I believe this should link to `userguide/inspecting_dependencies.html#sec:identifying_reason_dependency_selection` and probably that section, or another one, could be enhanced to document the `--single-path` flag.

Reply:
done